WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -6,6 +6,7 @@ import CoreData
     public let coreDataStore: WMFCoreDataStore
     private let userDefaultsStore: WMFKeyValueStore?
     private let developerSettingsDataController: WMFDeveloperSettingsDataControlling
+    private let userID: Int?
```

Comment:
@l-olson1214 For your other editing slide, you are passing in `username` through the `populateYearInReviewReportData` method instead of setting it as a property like you are here. Maybe we should do the same with `userID`, to keep it consistent? This way you don't have to send in userID upon instantiation, only when the population method is called (which is only when we really need it).

Reply:
Done!